### PR TITLE
Update 2018 copyright

### PR DESCRIPTION
**Pull request by xmr-eric**

### CODE REVIEW

**Comment by fluffypony** [comment]
@moneromooo-monero updating the date isn’t claiming copyright on behalf of the individual, and I’m not seeing any changes besides year changes on Monero Project copyright notices. In addition, as we discussed this time last year, we’ve been advised to update these notices annually on all files, not just on files we’ve changed, in order to make it clear that the copyright notice is current and in effect.

**Comment by moneromooo-monero** [comment]
Updating the date IS claiming you hold copyright of *something* in the file from that year. You are making a false representation of when the code in that flie is allowed to lapse into the public domain.

**Comment by moneromooo-monero** [comment]
Actually, the GNU recommendation is to blindly update all files anyway. That seems weird, but these guys are probably as clued in to the details of this as you'll find, so I've chaned my mind and this can go in despite the spam.

File: CMakeLists_IOS.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Portions Copyright (c) 2017, The Monero Project
+# Portions Copyright (c) 2018, The Monero Project
```
Comment (iDunk5400) [inline CMakeLists_IOS.txt]: IMO, this and others like this one should be updated to `2017-2018`.

**Comment by fluffypony** [comment]
@xmr-eric please rebase

**Comment by xmr-eric** [comment]
Fixed (I think).

**Comment by leonklingele** [comment]
Does not updating a copyright notice have any legal consequences? I'm not a lawyer but this patch seems superfluous to me.

**Comment by moneromooo-monero** [comment]
Putting it in source is a way to have better evidence/argument an infringer "knew or should have known".
I think updating is not needed technically, it's just good practice. I just don't like the spamming all over the place, as it feels sleazy to me (apparently wrongly, since FSF recommends it). In most countries, the Berne convention means you don't have to assert copyright explicitely to get it. It's just down to how a judge will rule AFAIK: the more arguments you put on your side, the likelier you are to win, it it comes to that.

**Pull request merged**